packages/api-tests/reporter.js

```diff
@@ -0,0 +1,62 @@
+var mocha = require('mocha');
+module.exports = MyReporter;
+const testContainer = process.env.MOCHA_CONTAINER;
+var fs = require('fs');
+var path = require('path');
+
+let jsonOutput = [];
+const pattern = /(#[\w.()]+[\s]?)+/;
```

Comment:
Actually, I think multiple tags will be necessary when we want specific container tests (i.e. `#electron-only`) so I think we should keep the space